examples/android/helloworld/app/CMakeLists.txt

```diff
@@ -100,8 +66,8 @@ add_library(helloworld_proto_lib
   SHARED ${HELLOWORLD_PROTO_HDRS} ${HELLOWORLD_PROTO_SRCS})
 
 target_link_libraries(helloworld_proto_lib
-  libprotobuf
-  libgrpc++
+  grpc++
+  ${_gRPC_PROTOBUF_LIBRARIES}
```

Comment:
nit:  the _gRPC_PROTOBUF_LIBRARIES is meant to be internal (hence the "_" prefix), so I'd suggest just referring to "libprotobuf" explicitly here.

Reply:
Done